packages/postcss-lowercase-text/src/__tests__/data/pseudo-elements.js

```diff
@@ -0,0 +1,171 @@
+  },
+  ':last-child': {
+    description:
+      'Matches any element that is the last child element of its parent.',
+  },
+  ':last-of-type': {
+    description:
+      'Matches the last sibling with the given element name in the list of children of its parent element.',
+  },
+  ':left': {
+    description: 'Matches any left page when printing a page.',
+  },
+  ':link': {
+    description: 'Matches links inside elements.',
+  },
+  ':not': {
+    argument: 'selector',
+    description: 'Matches an element that is not represented by the argument.',
+  },
+  ':nth-child': {
+    argument: 'an+b',
+    description:
+      'Matches an element that has an+b-1 siblings before it in the document tree.',
+  },
+  ':nth-last-child': {
+    argument: 'an+b',
+    description:
+      'Matches an element that has an+b-1 siblings after it in the document tree.',
+  },
+  ':nth-last-of-type': {
+    argument: 'an+b',
+    description:
+      'Matches an element that has an+b-1 siblings with the same element name after it in the document tree.',
+  },
+  ':nth-of-type': {
+    argument: 'an+b',
+    description:
+      'Matches an element that has an+b-1 siblings with the same element name before it in the document tree',
+  },
+  ':only-child': {
+    description: 'Matches any element which is the only child of its parent.',
+  },
+  ':only-of-type': {
+    description: 'Matches any element that has no siblings of the given type.',
+  },
+  ':optional': {
+    description:
+      'Matches any <input> element that does not have the required attribute set on it.',
+  },
+  ':out-of-range': {
+    description:
+      'Matches when an element has its value attribute outside the specified range limitations for this element.',
+  },
+  ':read-only': {
+    description: 'Matches when an element is not writable by the user.',
+  },
+  ':read-write': {
+    description:
+      'Matches when an element is editable by user like text input element.',
+  },
+  ':required': {
+    description:
+      'Matches any <input> element that has the required attribute set on it.',
+  },
+  ':right': {
+    description:
+      'Matches any right page when printing a page. It allows to describe the styling of right-side page.',
+  },
+  ':root': {
+    description:
+      'Matches the root element of a tree representing the document.',
+  },
+  ':scope': {
+    description:
+      'Matches the elements that are a reference point for selectors to match against.',
+  },
+  ':target': {
+    description:
+      'Matches the unique element, if any, with an id matching the fragment identifier of the URI of the document.',
+  },
+  ':valid': {
+    description:
+      "Matches any <input> or <form> element whose content validates correctly according to the input's type setting",
+  },
+  ':visited': {
+    description: 'Matches links that have been visited.',
+  },
+};
+
+export default pseudoElementsList;
```

Comment:
Can we write a script and get it from `mdn`?

Reply:
I don't know of any API to request from MDN !

can you share any link.